### PR TITLE
Juan Viguera Diez

### PR DESCRIPTION
**Pull request by JuanViguera**

### CODE REVIEW

File: assignments/students/juan_vigueradiez/reflection.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## What do you see as the limits for static analysis tools in ML? For code, models, and for data?
+I think the main limitations of static analysis for ML are related to the fact that is very difficult to predict (or even approximate) the actual behavior of a ML model without running its implementation. Indeed, we often we get surprised by the performance of these methods. Moreover, the performance is heavily dependent on the data that is used, complicating things even more. However, I think we can still benefit from statistical analysis for ML. Some options that come to mind are good practices and suggestions when designing and training models (for example suggesting using the state-of-the-art alternative when one option is clearly superior to the rest) or suggesting standard pre-processing procedures when available for the data we are using. Wrapping up, it is challenging to use statistic analysis for ML, but I still think there a lot potential in it, specially with integrated tools in development environments.
```
Comment (robertfeldt) [inline assignments/students/juan_vigueradiez/reflection.md:11]: Please check this, you probably mean "static analysis" and not "statistic"!?

**Comment by robertfeldt** [comment]
Minor comment on last question. Please check and then we can merge.

**Comment by JuanViguera** [comment]
Apologies for the missprint! Thanks for pointing it out. Should be fixed now.

**Comment by robertfeldt** [comment]
np, now good, you are done with assignment 1.

**Pull request merged**